ZuluContent/Gumps/PolymorphGump.cs

```diff
@@ -16,94 +16,118 @@ public PolymorphCategory(int num, params PolymorphEntry[] entries)
 
         public int LocNumber { get; }
     }
-
-    public class PolymorphEntry
+    public record PolymorphEntry
```

Comment:
@michaelwhelehan FYI I was going to keep the existing Polymorph gump but decided to keep the random functionality from ZH3.

Shape Shift had a gump associated with it so would make sense to repurpose this gump for that, see the comment I left in PolymorphSpell.cs on how to use it async.